dashboard/app/main.go

```diff
@@ -1126,12 +1142,21 @@ func loadManagers(c context.Context, accessLevel AccessLevel, ns, manager string
 		if now.Sub(mgr.LastAlive) > 6*time.Hour {
 			uptime = 0
 		}
+		// TODO: ideally we'd want to explicitly detect and mark cases, when
```

Comment:
What would we do differently for such builds?

Reply:
We'd add some note that attached the coverage report is related to the previous build, not to the current one. It should help the user understand what exactly they are downloading.

Reply:
I am not sure users care about current build/previous build. They may want to know the kernel commit, and I assume we could show it somehow for the report (but nobody asked for this).

Reply:
Users in general probably don't care, but I'd prefer to see if the coverage report is (already) worth looking at. Adjusted the comment.